ui/apps/everest/src/components/cluster-form/resources/resources.tsx

```diff
@@ -220,7 +226,7 @@ const ResourcesToggles = ({
             onChange: (_, value) => {
               if (value !== CUSTOM_NR_UNITS_INPUT_VALUE) {
                 resetField(customNrOfUnitsInputName, {
-                  keepError: false,
+                  keepError: true,
```

Comment:
why do we need to keep error?

Reply:
In case there is a custom value of `1` with error and we trigger toggle group, the error will be cleared. We need the field to keep an error state if the value is still `1`